src/table.ts

```diff
@@ -719,11 +719,10 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
    * region_tag:bigtable_api_table_readstream
    */
   createReadStream(opts?: GetRowsOptions) {
-    const options = opts || {};
+    const options: GetRowsOptions = opts || {};
     const maxRetries = is.number(this.maxRetries) ? this.maxRetries! : 10;
     let activeRequestStream: AbortableDuplex | null;
     let rowKeys: string[];
-    let filter: {} | null;
```

Comment:
Not used anymore.

Reply:
This doesn't affect at all what users pass in, right? If this doesn't affect what users pass in, then cool. If it does, we have to proceed with caution